<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkersDialog</class>
 <widget class="QDialog" name="WorkersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Workers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblList">
     <property name="text">
      <string>Workers:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTableWidget" name="tableWorkers">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pBtnAdd">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pBtnRem">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>160</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
